cves/CVE-2017-5648.yml

```diff
@@ -44,16 +44,24 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
+  While investigating bug 60718, it was noticed that some calls application
+  listeners did not use the appropriate facade oject. when running an untrusted
```

Comment:
Object is misspelled, "oject".  Other than that, this is a great description that is easy to understand.

Reply:
fixed